BioscoopSysteemAPI/BioscoopSysteemAPI/DTOs/PaymentDTOs/PaymentCreateDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace BioscoopSysteemAPI.DTOs.PaymentDTOs
+{
+    public class PaymentCreateDTO
```

Comment:
Same issue: verschillende PaymentDtos met dezelfde implementatie. wellicht kunnen we er 1 van maken en deze PaymentDto steeds weer inzetten. Wat denken jullie?